app/views/pages/robots.text.erb

```diff
@@ -6,5 +6,7 @@ Disallow: /connect/@*
 Disallow: /search?q=*
 Disallow: /search/?q=*
 Disallow: /listings*?q=*
+Disallow: /mod/*
+Disallow: /mod?*
```

Comment:
While we are here already, should we disallow things like `/internal` too?

Reply:
Good idea